tests/test_env.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+
+
+@pytest.fixture()
+def env_data():
+    with open('.env') as f:
+        lines = f.read().split('\n')
+        list_variable = []
+        for line in lines:
+            variable_name = line.split('=')[0]
+            list_variable.append(variable_name)
+
+    return list_variable
+
+
+@pytest.fixture()
+def env_example_data():
+    with open('.env.example') as f:
+        lines = f.read().split('\n')
+        list_variable = []
+        for line in lines:
+            variable_name = line.split('=')[0]
+            list_variable.append(variable_name)
+
+    return list_variable
+
+
+def test_variable_name_in_env_is_uppercase(env_data):
+    """check used name of variable are uppercase"""
+    for variable in env_data:
+        if variable.upper() == variable:
```

Comment:
Just write "assert variable.upper() == variable", also please don't use something like "variable" for variable name